docs/guides/BUILDING.md

```diff
@@ -40,6 +40,12 @@ synchronize submodules:
 git submodule update --init
 ```
 
+After synchronizing the modules you need to pull the latest changes:
```

Comment:
Pull the latest changes of what?  `submodule update --init` should be updating all the modules to the relevant SHAs...

If you are trying to document how to fully update a repo to a new SHA, that would be a `git pull` _followed_ by a submodule update?

Reply:
Ok if the git pull needs to be done before the sync, also ok (but the docs also do notstate this).

I can try that oncve the next PR got merged here on master and see if this works for me. I will then update here

Reply:
You're right that the docs don't say that, it being basic git.  Fixing them to say that would be a good idea.  I just don't want them giving the idea that you do pull after submodule update.  ;)

Reply:
All good ... I'm also just a "git user" and in fact it worked for me that way, biut as said I will try your approach and update the PR. Will set to draft till then

Reply:
Verified and Updated